antarest/study/business/output/variables_management.py

```diff
@@ -378,3 +386,49 @@ def _checks_variables_view_arguments_coherence(
         if any([thermal_id, renewable_id, st_storage_id]):
             raise OutputVariablesViewError(output_id, "You provided an renewable/thermal/storage id for areas")
         return AreaOutputIdentifier(area_id)
+
+
+def get_view_from_dataframe(dataframe: pd.DataFrame, variable_name: str) -> OutputVariablesView:
+    """Transform the given dataframe into a OutputVariablesView model"""
+    dataframe["idx"] = dataframe.groupby(MCYEAR_COL).cumcount()
+    df_pivot = dataframe.pivot(index="idx", columns=MCYEAR_COL, values=variable_name)
+    data = df_pivot.to_dict(orient="split")
+    del data["index"]
+    return OutputVariablesView.model_validate(data)
+
+
+def get_output_view_inside_db(
+    study_id: str,
+    output_id: str,
+    variable_type: OutputVariablesType,
+    variable_name: str,
+    area_id: str | None = None,
```

Comment:
Here again we'd better use the validate identifer instead of passing all arguments